### DISCUSSION TITLE
Line Art width based on distance?

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#436] [Q&A] [answered]
How do I make the width of the Line Art become thinner when an object or something is farther away from the view?

**Comment by VelocityRa** [comment, marked as answer]
You can do that with a setup like so:
![image](https://user-images.githubusercontent.com/6632271/199004220-03325e58-c36a-463d-bd66-e889598c4a80.png)

**Reply by VelocityRa** [reply]
Unfortunately the Line Width node seems to have a width cap, you can see ie. the closest Suzannes above should probably have more line thickness. Is this an unsolvable limitation of the line rendering approach used @pragma37?

**Reply by Psi25** [reply]
you can change the cap on the line renderer found in the default render layer

**Comment by pragma37** [comment]
The Line Width node has this as a built-in feature:
![imagen](https://user-images.githubusercontent.com/36610999/199347200-79336797-2aa6-4800-9aac-552a9807354c.png)
